src/libponyc/codegen/genlib.c

```diff
@@ -92,64 +94,11 @@ static bool reachable_actors(compile_t* c, ast_t* program)
     return false;
   }
 
+  printf("Selector painting\n");
```

Comment:
On purpose, to show compiler progress. We can change these messages, but I think as a separate PR.